src/switch-angular-builder.ts

```diff
@@ -0,0 +1,33 @@
+import {versions} from './angular-versions';
+import {AngularJSON} from './types/angular-json';
+
+const oldBuilder = '@angular-devkit/build-ng-packagr:build';
+const newBuilder = '@angular-devkit/build-angular:ng-packagr';
```

Comment:
suggestion: Make these names more useful, for example `pre10_1LibraryBuilder` and `the10_1LibraryBuilder`.